### PR TITLE
waypoint suggestions

### PR DESCRIPTION
**Pull request by egorchakov**
- add waypoints to `config/_templates/dataset/yaak.yaml`
- restrict waypoint specific stuff to a couple classes -- likely to be updated/generalized with e.g. `geopolars`

### CODE REVIEW

**Comment by koritsky** [comment]
It looks good for yaak waypoints, but i have some remarks regarding carla wpts.

F.ex , carla wpts are in x/y coordinates directly, hence no crs transform is needed
https://github.com/yaak-ai/rbyte/blob/805a61f48cc532219a572efb050e54d985430e22/src/rbyte/io/yaak/waypoints/normalizer.py#L59 

and again, timestamp and heading angles preprocessing are different for carla
https://github.com/yaak-ai/rbyte/blob/805a61f48cc532219a572efb050e54d985430e22/src/rbyte/io/yaak/waypoints/preprocessor.py#L32-L37

so my propose is to have separate preprocessor for each data type. F.ex. having
`rbyte.io.carla.waypoints.preprocessor.CarlaWaypointPreprocessor` would make no harm in my opinion.  Also it seems natural from the way files are organized, wdyt?

Maybe it would've been a better if i added carla config in #44 right away. Double work for us i wanted to avoid looks inevitable now

---
My suggestion:
1. Merge this into `feat/waypoints`
2. I will add carla waypoints config the way I see it
3. @egorchakov check everything again

File: src/rbyte/viz/loggers/rerun_logger.py [hunk 1]
```diff
@@ -177,10 +177,17 @@ def _build_columns(
...
                 case rr.Points3D.columns:
                     match (tensor := kwargs[key := "positions"]).shape:
-                        case (3,):
+                        case (2 | 3,):
+                            if tensor.shape == (2,):
+                                kwargs[key] = torch.cat([tensor, torch.zeros(1)])
                             return config.instantiate(**kwargs.cpu().numpy())  # pyright: ignore[reportUnknownMemberType]
 
-                        case (*batch_dims, n, 3):
+                        case (*batch_dims, n, 2 | 3):
```
Comment (koritsky) [inline src/rbyte/viz/loggers/rerun_logger.py:185]: So the point of this dirty hack is that simple `rr.Points2D` displays with y axis down and [currently there is no way to change it](https://github.com/rerun-io/rerun/issues/1387). Displaying in 3D with z=0 looks much better. 

alternating `y=-y` is confusing since it displays wrong values that've already caused me some headache in the past
